src/config/settings.py

```diff
@@ -219,7 +219,7 @@
 # This can be useful if your S3 buckets are public.
 AWS_QUERYSTRING_AUTH = False
 DEFAULT_FILE_STORAGE = os.getenv(
-    "DEFAULT_FILE_STORAGE", "storages.backends.s3boto3.S3Boto3Storage"
+    "DEFAULT_FILE_STORAGE", "django.core.files.storage.FileSystemStorage"
```

Comment:
I was checking Kubernetes config and I think this is not defined in the service configuration (at least in staging). Could it be using this default value? In that case I suppose we should set this value. Just checking if you're aware of this   🙂